### ISSUE TITLE
[lightning] Support multiple training loggers

### ISSUE THREAD

**Issue by karl-richter** [ourownstory/neural_prophet#830]
> Note: This feature depends on the introduction of Lightning as model wrapper.

**Desired functionality**
Aside from the custom MetricsLogger, additional loggers can be passed to the Lightning Trainer (eg. the TensorBoardLogger). However, when activating the feature with the custom MetricsLogger, the loggers seem to interfere and eg. mix-up the storage locations of the checkpoints and model hyperparameters. [Docs](https://pytorch-lightning.readthedocs.io/en/stable/extensions/logging.html)

**Current state**
- Functionality is implemented but commented out in the `configure_trainer()` function in `utils.py`.
- Loggers seem to interfere, eg. the TensorBoardLogger mixes up the storage locations of the model.hparams and checkpoint into two separate folders (which is not the case if only the MetricsLogger or the TensorBoardLogger are passed to the Trainer). Technically the Trainer supports multiple Loggers in parallel.

**Required changes**
Debug the behaviour and possibly extend the implementation of the MetricsLogger to avoid interference.

**Comment by karl-richter** [comment]
Update: the custom logger in the Lightning pr #837 now inherits from the TensorBoardLogger instead of the BaseLightningLogger. Thus,TensorBoard is now supported by default.

**Comment by karl-richter** [comment]
Since the TensorBoardLogger is supported by default, this issue is no longer needed.
